tests/test_cliCmdReport.py

```diff
@@ -71,7 +71,7 @@ def test_report(self):
 
         result = self.runner.invoke(
             pipetask_cli,
-            ["report", graph_uri, "-b", self.root, test_filename, "--no-logs"],
+            ["repo", self.root, "report", graph_uri, test_filename, "--no-logs"],
```

Comment:
You seem to have swapped the arguments here? It should be

``` ["report", self.root, graph_uri, test_filename, "--no-logs"]```

Reply:
Thank you thank you thank you!